docs/src/docs/asciidoc/index.adoc

```diff
@@ -19,6 +19,16 @@ If you are using alternative build systems, see <<alternative-build-systems.adoc
 [[changelog]]
 == Changelog
 
+=== Release 0.9.21
+
+==== Gradle plugin
+
+- TBD
```

Comment:
suggestion: remove this as it is not necessary that there will be Gradle fixes in this release